scripts/linters/python_linter.py

```diff
@@ -60,15 +60,15 @@ def write(self, message):
         """Writes the given message to messages list.
 
         Args:
-            message: str. Message returned by the pylinter.
+            message: str. Message returned by the method.
         """
         self.messages.append(message)
 
     def read(self):
-        """Returns the pylint messages as a list.
+        """Returns the output messages as a list.
 
         Returns:
-            list(str). Returns the pylint messages list.
+            list(str). Returns the output messages list.
```

Comment:
Returns the --> The

Perhaps a better phrasing: The list of output messages.

Reply:
Done.